src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_section_jump.scss

```diff
@@ -0,0 +1,132 @@
+/*doc
+---
+title: Section Jump
+name: section-jump
+category: Widgets
+---
+
+A module with a header, a left navigation of jump links and a center section
+of chapters. Often used fo navigating proposal details
+
+```html_example
+<div class="section-jump" id="section-jump-top">
+    <div class="section-jump-cover">
+        <h1 class="section-jump-cover-header">Section Jump header</h1>
+    </div>
+
+    <div class="section-jump-body">
+        <nav class="section-jump-navigation m-unnumbered" data-adh-sticky="">
```

Comment:
I think I would prefer not having a specific `section-jump-navigation`. Do you think it is possible to reuse the plain `jump-navigation` here?